aisynphys/cell_class.py

```diff
@@ -220,6 +220,8 @@ def classify_cells(cell_classes, cells=None, pairs=None):
         List of Cell instances to be classified.
     pairs : list | None
         List of pairs from which cells will be collected. May not be used with *cells*
+    missing_attr: 'ignore' | None
```

Comment:
How about 

    missing_attr : str
        Determines the behavior when a criteria attribute is missing on a 
        cell. If 'ignore', then the cell is excluded from the result,. If 'raise',
        then an exception is raised. Default is 'raise'.